Makefile

```diff
@@ -264,7 +264,7 @@ snapcraft-local.yaml: snapcraft-local.yaml.in
 # To avoid unintended conflicts with file names, always add to .PHONY
 # unless there is a reason not to.
 # https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
-.PHONY: build install install_debug dist clean distclean \
```

Comment:
This introduced a regression. `make build-linux` doesn't work correctly anymore...So I cannot run a local testnet which I do heavily.